docs/sources/v0.51.x/using-k6-browser/migrating-to-k6-v0-52.md

```diff
@@ -0,0 +1,215 @@
+---
```

Comment:
This page will be published under the current release (`v0.51`). Once this page has been approved, I'll work on copying this page, and making the necessary amendments to it for the next release (`v0.52`).